hr_expense_sheet_payment_state/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def is_invoice(self, include_receipts=False):
+        # override needed to ensure payment_state is computed
```

Comment:
```suggestion
        # Override move type check in the context of payment_state computation,
        # because the move linked to an expense sheet is of type entry.
```